.java-version

```diff
@@ -0,0 +1 @@
+17
```

Comment:
What is the exact syntax that we have to use here ? Also, which vendor version of the JDK is used here ? Does it refers to anything in the Elastic JVM catalog ?

Reply:
> What is the exact syntax that we have to use here ?

In this case, only major versions.

> which vendor version of the JDK is used here ?

For GitHub actions, it uses
- `temurin`, see https://github.com/elastic/elastic-otel-java/blob/091edb229a7b2c6baf90331c9a605265dbeb98d8/.github/workflows/gradle-goal/action.yml#L14

For Buildkite, it uses:
- https://jvm-catalog.elastic.co/jdk and more precisely [openjdk](https://github.com/elastic/elastic-otel-java/blob/091edb229a7b2c6baf90331c9a605265dbeb98d8/.buildkite/hooks/prepare-common.sh#L26C10-L28)

Reply:
Nevermind, I found the answer is included in this PR, so it's effectively using the "latest openjdk from Elastic JVM catalog".

```
  JAVA_URL=https://jvm-catalog.elastic.co/jdk
  JAVA_PKG="${JAVA_URL}/latest_openjdk_${JAVA_VERSION}_linux.tar.gz"
```